Makefile

```diff
@@ -21,6 +22,7 @@ define test_cover_only
 	@echo "# running unit tests with coverage analysis"
 	@go test -covermode=count -coverprofile=${COVER_DIR}coverage_unit1.out ./logging/logrus
 	@go test -covermode=count -coverprofile=${COVER_DIR}coverage_unit2.out ./db/keyval/etcdv3
+	@go test -covermode=count -coverprofile=${COVER_DIR}coverage_unit2.out ./db/keyval/redis
```

Comment:
pleasue use unique filename such as `${COVER_DIR}coverage_unit7.out` and add the file into the list on the line 32